.github/renovate.json

```diff
@@ -1,5 +1,10 @@
 {
-  "extends": ["config:base", ":preserveSemverRanges", "schedule:weekly"],
+  "extends": [
+    "config:base",
+    "pinOnlyDevDependencies",
```

Comment:
no need for this

Reply:
Just wanted to keep things in sync. I don't mind. I can remove it.

Reply:
Exactly, our OSS repos don't use this anymore 😉
Thanks!